hazelcast/src/main/java/com/hazelcast/query/impl/BitmapIndexStore.java

```diff
@@ -145,6 +147,7 @@ public void insert(Object value, QueryableEntry entry, IndexOperationStats opera
                 }
 
                 bitmap.insert(values, key, entry);
+                accountEntryInsert(operationStats, statValues);
```

Comment:
can we account stats after releasing lock? If yes, we can put `accountEntryInsert` in the end of the method after releasing write locks.

Reply:
@taburet proposed to pass `operationStats` instance to `Bitmap` methods. 
I reviewed my approach and agreed, that Sergey's proposition makes sense and would be cheaper.

Reply:
it will also work, my concern was to do minimum work inside critical section(if it is valid to calculate stats after lock-release)

Reply:
Sure, got you.